src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_5001_6000.cpp

```diff
@@ -1509,6 +1509,21 @@ GUI_TEST_CLASS_DEFINITION(test_5417) {
     CHECK_SET_ERR(!lt.hasErrors(), "Errors in log: " + lt.getJoinedErrorString());
 }
 
+GUI_TEST_CLASS_DEFINITION(test_5421) {
+    // 1. Build doplot for "data/samples/Genbank/murine.gb" and "data/samples/Genbank/sars.gb".
+    // 2. Remove sars.gb from project
+    // Expected state: Save dialog appeared.
+    GTUtilsDialog::waitForDialog(os, new DotPlotFiller(os));
+    Runnable *buildDotPlot = new BuildDotPlotFiller(os, dataDir + "samples/Genbank/sars.gb", dataDir + "samples/Genbank/murine.gb");
```

Comment:
Do not use local variables for fillers. Filles do not survive the waitForDialogCall().